### PR TITLE
[Insights Management] Add `Customize your insights` card

### LINKED ISSUE #12244: [Insights Management] Add `Customize your insights` card
Add `Customize your insights` card to the top of the Insights table.
- It appears at the top of the insights tab and remains there until it has been interacted with. 
- It should appear at the top of insights on every site but once you interact with it in one place it would disappear everywhere.
- `Later` hides the card permanently.
- `Try it now` opens the `Add New Stats Card` view.

From Zeplin:

customize_card

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #12244 

To test:

---
TIP: If you'd like to force the Customize card to show without reinstalling the app: 
- In `SiteStatsInsightsTableViewController:loadInsightsFromUserDefaults`, set `hideCustomizeCard` to `false`.
- When you navigate away from Stats and back, the card will show.

NOTE: The image is currently using 1/2/3x PNGs. I've asked @SylvesterWilmott for a PDF version, and will update it when received.

---
- Go to Insights.
- Verify the customize card is displayed at the top.

customize

---
- Select 'Try it now'.
- Verify an alert is displayed (it's temporary, since the IM view doesn't exist yet).

alert

---
- Select 'Dismiss'.
- Verify the Customize card is dismissed.
  - NOTE: I banged my head for a while trying to animate that. I'll come back to it later.
- Verify the Customize card is not displayed again.
  - Navigate away from Stats and return.
  - Select another site.
  - Relaunch the app.

---
- Reinstall the app.
- Verify the Customize card is displayed.

---
- Disable the `statsInsightsManagement` feature flag.
- Verify the Customize card is not displayed.

---
Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott - screenshots:

iphone_portrait

ipad_portrait

ipad_landscape

**Comment by SylvesterWilmott** [comment]
This looks great!

Since you asked me if we could replace this image with a .pdf version, would it also be feasible to add a tablet version as well which would only show on iPads?

File: WordPress/Classes/ViewRelated/Stats/Insights/Insights Management/CustomizeInsightsCell.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+class CustomizeInsightsCell: UITableViewCell, NibLoadable {
+
+    // MARK: - Properties
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var contentLabel: UILabel!
+    @IBOutlet weak var dismissButton: UIButton!
+    @IBOutlet weak var tryButton: UIButton!
+    @IBOutlet weak var bottomSeparatorLine: UIView!
+
```
Comment (danielebogo) [inline WordPress/Classes/ViewRelated/Stats/Insights/Insights Management/CustomizeInsightsCell.swift]: You can remove the _weak_ here and make them private if you don't need to expose these properties.

**Review by danielebogo — CHANGES_REQUESTED** [review]
Hey @ScoutHarris I left one tiny comment, other than that it looks good. I tested it and it worked perfectly. Also I saw this warning building the branch: `CustomizeInsightsCell.xib:w8p-Zf-FRY: warning: Automatically Adjusts Font requires using a Dynamic Type text style [9]`.

A part from these 2 small things everything is good to :shipit:

**Comment by ScoutHarris** [comment]
Hey @SylvesterWilmott .

> would it also be feasible to add a tablet version as well which would only show on iPads?

Yep, that's totally doable.

**Comment by ScoutHarris** [comment]
Hey @danielebogo . Thanks for catching those issues. Both have been fixed. Ready for another review please.

**Review by danielebogo — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**